.travis.yml

```diff
@@ -11,6 +11,9 @@ env:
   - 'DATABASE_URL=sqlite:///:memory:'
   - DATABASE_URL=postgresql+psycopg2://postgres@127.0.0.1/dataset
   - DATABASE_URL=mysql+pymysql://root@127.0.0.1/dataset?charset=utf8
+  - DEACTIVATE_NUMPY=1
+  - DEACTIVATE_NUMPY=0
```

Comment:
Intentionally declaring the same variable key twice?

Reply:
If you want to let Travis CI test both when numpy is enabled and when numpy is not enabled. Don't you need both? I'm open for suggestions/changes.

Reply:
I checked this and indeed your assumption was indeed correct! This will generate builds for EACH environment k/v defined under `env`. Not the behaviour I expected 😄